lib/team_api/cross_referencer.rb

```diff
@@ -3,6 +3,8 @@
 require_relative 'api'
 require_relative 'canonicalizer'
 
+require 'lambda_map_reduce'
+
 module TeamApi
```

Comment:
Looks like there are 3 classes in this file - thoughts on breaking them into 3 files, each named after the class? 

When i saw the changes above (`CrossReferencer.map_reduce` -> `LambdaMapReduce.map_reduce`) I assumed the `CrossReferencer` class was in this file...and it was, but 2 classes were above it. I think it would be easier to find these classes if they were in separate files named after them rather than all together.

Thoughts?